clocks-impl.cc

```diff
@@ -19,6 +19,21 @@
  * along with Scylla.  If not, see <http://www.gnu.org/licenses/>.
  */
 
+#include <seastar/core/print.hh>
+
+#include "db_clock.hh"
+#include "timestamp.hh"
+
 #include "clocks-impl.hh"
 
 std::atomic<int64_t> clocks_offset;
+
+std::ostream& operator<<(std::ostream& os, db_clock::time_point tp) {
+    auto t = db_clock::to_time_t(tp);
+    return os << std::put_time(std::gmtime(&t), "%Y/%m/%d %T");
+}
+
+std::string print_ts(api::timestamp_type ts) {
```

Comment:
s/ts/timestamp/, let's not introduce more acronyms.

Reply:
Now it's `format_timestamp`